src/language-css/parser-postcss.js

```diff
@@ -438,32 +439,32 @@ function parseNestedCSS(node) {
 }
 
 function parseWithParser(parser, text) {
+  const frontmatter = (function() {
+    const parsed = parseFrontmatter(text);
+    text = parsed.content;
+    return parsed.frontmatter;
+  })();
```

Comment:
You should be able to remove the IIFE here.

Reply:
I know that, but why?

Reply:
It feels like it would run slower, And it makes the code more complex and harder to understand.